src/App.tsx

```diff
@@ -1,9 +1,31 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
+function filtered(title: string, substring: string) {
```

Comment:
Consider renaming `filtered` function to a more descriptive name, like `isMatch` or `containsSubstring`. 🤔